### PR TITLE
Release 3.2.0

### PR DESCRIPTION
**Pull request by dhollinger**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,168 +1,259 @@
...
-- Include jira::facts class by default.
-- Adding new feature generating a content.xml configuration
-- Add a notify and subscribe options to the jira service
-- Provide ability to set the bind address of Tomcat via the jira::tomcatAddress parameter
-- Bump jira version to 6.4.1
-
-Thanks to Scott Searcy, Oliver Bertuch, Paul Geringer, Eric Shamow, William Lieurance, Doug Neal for their contributions.
-
-
-##2014-01-21 - Release 1.1.5
```
Comment (bastelfreak) [inline CHANGELOG.md]: every release up to 1.2.0 is missing in the new CHANGELOG.md. Those entries should be moved into a HISTORY.md with an adjusted title (it has to match the style from the github-changelog-generator one). Afterwards regenerate the CHANGELOG.md.

File: HISTORY.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
+# Changelog
```
Comment (alexjfisher) [inline HISTORY.md]: Remove this line.

File: HISTORY.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+## 2017-01-13 - Release 3.1.0
```
Comment (alexjfisher) [inline HISTORY.md]: For github changelog generator to detect this version properly, change it to...
```
## v3.1.0 (2017-01-13)
```

File: HISTORY.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+## v1.1.2 (2014-10-19)
+- Add new parameter: jvm_permgen, defaults to 256m.
+- Updates to readme
+
+
+## v1.1.1 (2014-10-11)
+- Fix typo of in module nanliu-staging, preving module from being installed
+
+
+##2014-10-09 - Release 1.1.0
```
Comment (alexjfisher) [inline HISTORY.md]: Could you fix this markdown?

**Pull request merged**